packages/table/src/headers/_headers.scss

```diff
@@ -51,7 +51,20 @@
   }
 }
 
-.bp-table-th-menu {
+.bp-table-th-menu-container {
+  flex-shrink: 0;
```

Comment:
shouldn't need flex properties if you're using `position: absolute` but whatever